maven/bnd-maven-plugin/src/test/resources/integration-test/test/verify.groovy

```diff
@@ -122,6 +130,10 @@ assert wrapper_jar.getEntry('lib/osgi.annotation.jar') != null
 assert tests_test_jar.getEntry('org/example/test/') != null
 assert tests_test_jar.getEntry('org/example/impl/') == null
 assert tests_test_jar.getEntry('org/junit/') != null
+assert war_jar.getEntry('WEB-INF/') != null
+assert war_jar.getEntry('WEB-INF/classes/org/example/impl/') != null
+assert war_jar.getEntry('WEB-INF/lib/test.api.bundle-0.0.1.jar') != null
+assert war_jar.getEntry('WEB-INF/lib/osgi.annotation-6.0.1.jar') == null
```

Comment:
Should you also test that osgi.cmpn (provided scope) is also NOT present?

Reply:
sure I can add that.

Reply:
fixed